### PR TITLE
eliminate body wrapper

### PR DESCRIPTION
**Pull request by ryan-mahoney**
#### Summary of changes
**Asana Ticket:** [🐞 Safari | Website doesn't render correctly](https://app.asana.com/0/555089885850811/1139044049600985)

WIP

<br>
Assigned to: @NAME

### CODE REVIEW

File: apps/site/assets/css/_layout-core-shared.scss [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
-    overflow: hidden;
-  }
-}
+// .body-wrapper {
+//   display: flex;
+//   flex: 0 0 auto;
+//   flex-direction: column;
+//   height: 100%;
+//   max-width: 100%;
+//   overflow: visible;
```
Comment (meagonqz) [inline apps/site/assets/css/_layout-core-shared.scss]: fwiw I was wondering if this issue had to do with a change we did from `overflow: auto` to `overflow: visible` after adding the modal.

iOS mobile Safari really is a troublesome tenant... i.e. this whole spiel: https://www.eventbrite.com/engineering/mobile-safari-why/

Comment (ryan-mahoney) [reply apps/site/assets/css/_layout-core-shared.scss]: lol I read that article too

**Comment by ryan-mahoney** [comment]
is this PR still relevant since we found the cause?

**Comment by meagonqz** [comment]
Nope

**Pull request closed without merging**